zeppelin-web/app/scripts/app.js

```diff
@@ -95,9 +90,11 @@ angular
     'puElasticInput',
     'xeditable'
   ])
-  .filter('breakFilter', function() {
+  .filter('breakFilter', function () {
```

Comment:
right. it's not up to me to decide, so don't worry. it's just a one man's opinion here. we gladly accept contributions. there are so many unsolved problems in zeppelin; there is a lot that could be done :)